sdk/core/System.ServiceModel.Rest/src/Temporary/ModelSerializationExtensions.cs

```diff
@@ -0,0 +1,347 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.Json;
+using System.Xml;
+
+namespace System.ServiceModel.Rest.Shared.Core.Serialization
+{
+    public static class ModelSerializationExtensions
```

Comment:
I don't think we want to ship public extensions classes.  All of these come in from internal shared source used by the generator, and I think we can rework it so we don't need extensions types, but it will mean reworking the model serialization implementations beyond a simple type name replacement.  This is clearly to be done later, and maybe we can do it as part of the model serialization work in Ge?  @pallavit, @jsquire, can we? 🙂